common/tools/dev-tool/src/commands/samples/tsToJs.ts

```diff
@@ -20,7 +22,7 @@ export const commandInfo = makeCommandInfo(
 );
 
 const prettierOptions: prettier.Options = {
-  ...(require("../../../../eslint-plugin-azure-sdk/prettier.json") as prettier.Options),
+  ...(defaultPrettierOptions as prettier.Options),
```

Comment:
I do not think this will work. I believe we need the dynamic behavior of require. /cc @witemple-msft